src/gflownet/utils/conditioning.py

```diff
@@ -102,19 +104,20 @@ def sample(self, n):
         elif self.cfg.preference_type == "seeded":
             preferences = torch.tensor(self.seeded_prefs).float().repeat(n, 1)
         elif self.cfg.preference_type == "dirichlet_exponential":
-            a = np.random.dirichlet([1] * self.num_objectives, n)
+            a = np.random.dirichlet([self.cfg.preference_param] * self.num_objectives, n)
             b = np.random.exponential(1, n)[:, None]
             preferences = Dirichlet(torch.tensor(a * b)).sample([1])[0].float()
         elif self.cfg.preference_type == "dirichlet":
-            m = Dirichlet(torch.FloatTensor([1.0] * self.num_objectives))
+            m = Dirichlet(torch.FloatTensor([self.cfg.preference_param] * self.num_objectives))
             preferences = m.sample([n])
         else:
             raise ValueError(f"Unknown preference type {self.cfg.preference_type}")
         preferences = torch.as_tensor(preferences).float()
         return {"preferences": preferences, "encoding": self.encode(preferences)}
 
     def transform(self, cond_info: Dict[str, Tensor], flat_reward: Tensor) -> Tensor:
-        scalar_logreward = (flat_reward * cond_info["preferences"]).sum(1).clamp(min=1e-30).log()
+        # NO LOG NO LOG NO LOG NO LOG NO LOG NO LOG
```

Comment:
FYI we changed the way we handle this in #117 (which fixes this)

Reply:
I merged the main branch, will run tests tomorrow to check that everything is still working.